contrib/ci/check-source.py

```diff
@@ -202,6 +202,8 @@ def _test_line_blocked_fns(self, line: str) -> None:
             "g_assert(": "Use g_set_error() or g_return_val_if_fail() instead",
             "g_udev_device_get_sysfs_attr(": "Use fu_udev_device_read_sysfs() instead",
             "g_udev_device_get_property(": "Use fu_udev_device_read_property() instead",
+            "HIDIOCSFEATURE": "Use fu_hidraw_device_set_feature() instead",
+            "HIDIOCGFEATURE": "Use fu_hidraw_device_get_feature() instead",
```

Comment:
How about instead detecting ANY IOCTL call?  Presumably this PR is because of emulation, and that will protect us from any mistakes in hardcoded ioctl calls showing up that we miss.